File: proxy/java/IOrderableMask.java
package Proxy;

// Sipariş işlemlerini gerçekleştiren sınıflar tarafından uygulanan arayüz.
// UML diyagramındaki Service arayüzüne denk gelmektedir.
interface IOrderableMask {
    void createOrder(Person person);
}

File: proxy/kotlin/IOrderableMask.kt
package Proxy

// Sipariş işlemlerini gerçekleştiren sınıflar tarafından uygulanan arayüz.
// UML diyagramındaki Service arayüzüne denk gelmektedir.
internal interface IOrderableMask {
    fun createOrder(person: Person)
}